app/paramParser.js

```diff
@@ -29,6 +35,50 @@ function parseProcessor(key) {
     }
 }
 
+function parseBlur(key){
+    var splitArray = key.split('/');
+    var regex = /blur:(([0-9]*[.])?[0-9]+)/;
+    var regexMatch = splitArray[0].match(regex);
+    var blur = null;
+
+    if (regexMatch && regexMatch.length > 0) {
+        splitArray.splice(0, 1);
+
+        var blur = parseFloat(regexMatch[1])
+        blur = (blur < minBlurRadius) ? minBlurRadius : blur ;
+        blur = (blur > maxBlurRadius) ? maxBlurRadius : blur ;
+    }
+
+    var path = splitArray.join('/');
+
+    return {
+        blur: blur,
+        path: path
+    }
+}
+
+function parseAutoRotate(key){
+    var splitArray = key.split('/');
+    var regex = /auto_rotate:(true|false)/;
+    var regexMatch = splitArray[0].match(regex);
+    var auto_rotate = true;
+
+    if (regexMatch && regexMatch.length > 0) {
+        splitArray.splice(0, 1);
+        var rotate = JSON.parse(regexMatch[1]);
+        if (rotate == false) {
```

Comment:
this can just be (!rotate)

Reply:
or even 
just a single line...

auto_rotate = !JSON.parse(regexMatch[1]) if value is undefined as well then also will set this to true.. 

Also do you think JSON.parse ? is a good approach... can we just not do

auto_rotate = !(regexMatch[1].toLowerCase() == "false") or something like this.